src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -77,13 +77,19 @@ class NexusPublishPlugin : Plugin<Project> {
             val closeAndReleaseTask = project.tasks.register<Task>(
                     "closeAndRelease${capitalizedName}StagingRepository")
             closeTask {
+                setDescription("Closes open staging repository in '$name' Nexus instance.")
+                setGroup(PublishingPlugin.PUBLISH_TASK_GROUP)
                 mustRunAfter(initializeTask)
             }
             releaseTask {
+                setDescription("Released closed staging repository in '$name' Nexus instance.")
+                setGroup(PublishingPlugin.PUBLISH_TASK_GROUP)
```

Comment:
Please, do not do that. The tasks in the plugin are low-level, and adding group/description does pollute the output of `gradlew tasks` with very little gain.

Please try to keep only the very usable tasks in the default `gradlew tasks` output.

Reply:
In practice, making a release often includes "create and push Git tag", so I would expect the users would have their own wrappers over `close`, `release repository` and the other tasks.

It would be great if you could provide a plugin variation that does not pollute `gradlew tasks` output.
Of course, I can hide the tasks in my own plugin, however, it is sad to track the required tasks to hide :-/